app/assets/javascripts/experiments/table.js

```diff
@@ -456,7 +464,7 @@ var ExperimnetTable = {
   },
   clearRowTaskSelection: function() {
     this.selectedMyModules = [];
-    $('.select-all-checkboxes .sci-checkbox').prop('checked', false);
+    $('.select-all-checkboxes, .sci-checkbox').prop('checked', false);
```

Comment:
well, this line before was correct because it is only for removing the check from the checkbox at the header.
I think you need to add another line to be sure that you uncheck only the checkbox in the table. Not all checkboxes are on the page.

Reply:
Hi @aignatov-bio,
My bad!
From the naming, I thought it would clear only one row.

I suggest something like this:
- Rename the method to `clearSelectAllCheckboxes`
- Bring the selector back to ` $('.select-all-checkboxes .sci-checkbox').prop('checked', false);`
- Add another method that unchecks and updates a specific checkbox with the id `clearMyModuleSelector(myModuleId)`

What do you think?